src/client.h

```diff
@@ -37,5 +37,5 @@ void client_set_hostname_maxlen(int i);
 int client_handshake(int dns_fd, int raw_mode, int autodetect_frag_size,
 		     int fragsize);
 int client_tunnel(int tun_fd, int dns_fd);
-
+int handshake_check_v6(int tun_fd);
```

Comment:
this is an internal function, can you move it so it doesn't need to be declared here?